app/views/shared/_spotlight_search.html.erb

```diff
@@ -0,0 +1,36 @@
+<% main_resource = request.path.split("/").last.presence_in(%w[talks speakers events]) || "talks" %>
+<% resources = %w[talks speakers events] %>
+
+
+<%= ui_modal position: :top, close_button: false, data: {action: "keydown.meta+k@window->modal#open spotlight@window->modal#open"}, class: "spotlight"  do %>
```

Comment:
I'm a Mac user, so I can't test it. How would a Windows/Linux user use the hotkey?

Reply:
I am not a specialist but I think the `meta` key is cross platform https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/metaKey

on a related topic in a perfect world the hint for the hotkey should be plaform specific. Not sure I want to go that fare in this PR 😄 

![CleanShot 2024-11-15 at 16 02 06@2x](https://github.com/user-attachments/assets/838a6176-778e-4b99-a32e-e7bbc05c7eb9)